dot/sync/syncer_integeration_test.go

```diff
@@ -124,24 +121,23 @@ func newTestSyncer(t *testing.T) *Service {
 	cfg.BlockState.StoreRuntime(cfg.BlockState.BestBlockHash(), instance)
 
 	cfg.BlockImportHandler = new(mocks.BlockImportHandler)
-	cfg.BlockImportHandler.(*mocks.BlockImportHandler).On(
-		"HandleBlockImport", mock.AnythingOfType("*types.Block"), mock.AnythingOfType("*storage.TrieState")).
-		Return(func(block *types.Block, ts *rtstorage.TrieState) error {
-			// store updates state trie nodes in database
-			if err = stateSrvc.Storage.StoreTrie(ts, &block.Header); err != nil {
-				logger.Warnf("failed to store state trie for imported block %s: %s", block.Header.Hash(), err)
-				return err
-			}
-
-			// store block in database
-			err = stateSrvc.Block.AddBlock(block)
-			require.NoError(t, err)
-
-			stateSrvc.Block.StoreRuntime(block.Header.Hash(), instance)
-			logger.Debugf("imported block %s and stored state trie with root %s",
-				block.Header.Hash(), ts.MustRoot())
-			return nil
-		})
+	cfg.BlockImportHandler.(*mocks.BlockImportHandler).On("HandleBlockImport", mock.AnythingOfType("*types.Block"),
+		mock.AnythingOfType("*storage.TrieState")).Return(func(block *types.Block, ts *rtstorage.TrieState) error {
```

Comment:
keeping it as it was

```suggestion
	cfg.BlockImportHandler.(*mocks.BlockImportHandler).On(
		"HandleBlockImport", mock.AnythingOfType("*types.Block"), mock.AnythingOfType("*storage.TrieState")).
		Return(func(block *types.Block, ts *rtstorage.TrieState) error {
```

Reply:
Update.